src/vs/editor/contrib/codeAction/common/types.ts

```diff
@@ -193,6 +193,7 @@ export class CodeActionItem {
 	constructor(
 		public readonly action: languages.CodeAction,
 		public readonly provider: languages.CodeActionProvider | undefined,
+		public toMark?: boolean,
```

Comment:
Is there a more descriptive name for this? Maybe instead of `toMark`, pass in a `highlightRange`?